packages/components/yarn.lock

```diff
@@ -12488,3 +12488,8 @@ yargs@^7.0.0:
     which-module "^1.0.0"
     y18n "^3.2.1"
     yargs-parser "^5.0.0"
+
+yarn@^1.22.0:
```

Comment:
If yarn is removed from the package.json dependencies this should go away.

Reply:
great, thanks!